pkg/asset/machines/openstack/machinesets.go

```diff
@@ -32,23 +32,19 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 		total = int32(*pool.Replicas)
 	}
 
-	numOfAZs := int32(len(mpool.Zones))
-	// TODO(flaper87): Add support for availability zones
 	var machinesets []*clusterapi.MachineSet
 
-	for idx, az := range mpool.Zones {
-		replicas := int32(total / numOfAZs)
-		if int32(idx) < total%numOfAZs {
-			replicas++
-		}
```

Comment:
By using `Replicas: &total` on line 63, aren't we setting the total number of requested machines on EACH MachineSet? If I'm not mistaken, this means that if you set `replicas: 3` in install-config and 3 zones, you get a total of 9 machines spread across 3 zones

Reply:
I decided to remove this part from here, because it will be addressed in https://github.com/openshift/installer/pull/4009